src/rgw/rgw_reshard.cc

```diff
@@ -325,42 +315,31 @@ int RGWBucketReshard::clear_index_shard_reshard_status(rgw::sal::RGWRadosStore*
   return 0;
 }
 
-static int create_new_bucket_instance(rgw::sal::RGWRadosStore *store,
+static int update_num_shards(rgw::sal::RGWRadosStore *store,
```

Comment:
lets call this `set_target_layout()` since it's where the target_layout is initialized and written

Reply:
done